config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :articles, except: [:edit, :update]
+  resources :users, except: [:index]
+  resource :sessions, except: [:index, :edit, :update, :show]
```

Comment:
using `only` will make this slightly shorter and perhaps easier to read: `only: [:new, :create, :destroy]`. or is it `:delete`? I can never remember...

Reply:
It's :destroy haha